### ISSUE TITLE
Upgrading to Pieces.os 7.2.2 all my data is gone

### ISSUE THREAD

**Issue by furier** [pieces-app/support#41]
### Software

Desktop Application

### Operating System

Windows

### Bug details

Starting my IDE I got a prompt to upgrade my Pieces OS, so I did and afterwards when i started Pieces for Developers it started the new user onboard wizard where i had to log in again. 

To my surprise **all my snippets were gone** and I could remember a few days back that on my work pc **OneDrive** told me an unusual amount of files was deleted in one go and if i wanted to go ahead with it. I did as I had just upgraded pieces there as well (tho running 7.1.1) and I could see that the files being deleted were some pieces folders and I figured it was part of the update and everything worked. (and still does)

In an attempt to recover my snippets I logged onto my work pc via remote desktop and and zipped the Documents/com.pieces.os folder and restored it on my home computer, which I dont think was a good idea after learning the version difference. 7.1.1 vs 7.2.2 locally.

First time starting up Pieces OS after the file restoration the UI greeted me with a permission exception, but now it doesn't and things seem to work but my snippets are still gone. 🤦‍♂️

**Need some help here restoring my snippets please 🙋‍♂️**

**Comment by Frankie-B** [comment]
This just happened to me.  Everything is just gone.

**Comment by rosie-at-pieces** [comment]
Duplicate of https://github.com/pieces-app/support/issues/38

**Comment by furier** [comment]
@rosie-at-pieces I dont see how that helps me recover my lost data?

**Comment by mason-at-pieces** [comment]
Hey @furier! The issue we linked here will be the main issue for the recover snippets issue as well as another issue around denied access.

We are working on both and they are highly correlated so we combined them into 1 issue on our repo.

We should have a v7.2.4 patch for Pieces OS released in the next hour which will hopefully solve both of those issues and return your snippets and data as well!
